### PR TITLE
Adduser modal width Fix

### PR DESCRIPTION
**Pull request by Khaalid-oh**
## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have tested on Chrome and Firefox
- [ ] I have tested on a mobile device
- [ ] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

https://github.com/stakwork/sphinx-tribes/assets/84248666/b5c4bda7-f4fd-4a47-83fd-d6aeebc6e1ab

### CODE REVIEW

**Comment by ecurrencyhodler** [comment]
@Khaalid-oh thanks for submitting this. One of the acceptance criteria is to add component tests. Please add that to this PR.

**Comment by Khaalid-oh** [comment]
> @Khaalid-oh thanks for submitting this. One of the acceptance criteria is to add component tests. Please add that to this PR.

Oh, I've done that before. Let me add it to this PR

**Comment by ecurrencyhodler** [comment]
Okay let's do a code review.

File: frontend/app/yarn.lock
Comment (kevkevinpal) [inline frontend/app/yarn.lock:1]: please remove this from the diff you can do so by running
`git checkout origin/master -- frontend/app/yarn.lock`

File: frontend/app/src/config/ModeDispatcher.tsx
Comment (kevkevinpal) [inline frontend/app/src/config/ModeDispatcher.tsx:1]: please remove this from the diff you can do so by running
`git checkout origin/master -- frontend/app/src/config/ModeDispatcher.tsx`

**Pull request closed without merging**